tests/test_benchmark_cli.py

```diff
@@ -328,3 +330,82 @@ def test_format_is_json(f):
 )
 def test_format_is_cbor(f):
     assert format_is_cbor(f.value) is True
+
+
+def test_output_result_table_none():
+    test_table = [['field1', 'field2'], [1, 2], [3, 4]]
+    results_output = None
+    res = output_result_table(results_output, test_table)
+    # Should print as expected and return None
+    assert res is None
+
+
+def test_output_result_table_output():
+    test_table = [['field1', 'field2'], [1, 2], [3, 4]]
+    results_output = 'test'
+    res = output_result_table(results_output, test_table)
+    # Cleans up the output file generated
+    if os.path.exists(results_output):
+        os.remove(results_output)
+    assert simpleion.dumps(res, binary=False) == \
+           "$ion_1_0 [{field1:1,field2:2},{field1:3,field2:4}]"
```

Comment:
Let's use ion equivalence for these assertions, like

`assert_ion_equals(res, simpleion.loads("[{field1:1,field2:2},{field1:3,field2:4}]")`

Reply:
Added an assertion helper. Since they are Ion string, compare them after `simpleion.load()`.
```.py
def assert_ion_string_equals(act, exp):
    assert ion_equals(simpleion.loads(act), simpleion.loads(exp))
```